demo/client/src/view.ts

```diff
@@ -197,6 +202,60 @@ export abstract class DemoView extends PathfinderView implements RenderContext {
 
         this.atlasRenderingTimerQuery = null;
         this.compositingTimerQuery = null;
+
+        this.inVRRAF = false;
+        this.inVR = false;
+        this.vrDisplay = null;
+        if ("VRFrameData" in window) {
+           this.vrFrameData = new VRFrameData;
+        } else {
+            this.vrFrameData = null;
+        }
+
+        this.vrSetup();
+    }
+
+    vrSetup(): void {
+        if (navigator.getVRDisplays) {
+            navigator.getVRDisplays().then((displays) => {
+              if (displays.length > 0) {
```

Comment:
Usually I use early-return to reduce rightward drift, so I'd write this as:

    if (displays.length === 0)
        return;